src/main/java/frc/robot/maps/TempestMap.java

```diff
@@ -1,32 +1,121 @@
 import com.chopshop166.chopshoplib.outputs.EncodedSpeedController;
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.AnalogTrigger;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.Victor;
+
 @RobotMapFor("Tempest")
 public class TempestMap extends RobotMap {
 
     @Override
     public DifferentialDriveMap getDriveMap() {
         return new DifferentialDriveMap() {
+            @Override
+            public EncodedSpeedController getLeft() {
+
+                SendableSpeedController motors =  SendableSpeedController.wrap(new SpeedControllerGroup(new WPI_TalonSRX(4), new WPI_TalonSRX(1)));
+                return EncodedSpeedController.join(motors,new MockEncoder());
+
+
+            }
 
             @Override
             public EncodedSpeedController getRight() {
-                final SendableSpeedController rightGroup = SendableSpeedController.group(new WPI_TalonSRX(2),
-                        new WPI_TalonSRX(3));
-                return EncodedSpeedController.join(rightGroup, new MockEncoder());
+                SendableSpeedController motors = SendableSpeedController
+                        .wrap(new SpeedControllerGroup(new WPI_TalonSRX(2), new WPI_TalonSRX(3)));
+
+                return EncodedSpeedController.join(motors,
+                        new MockEncoder());
             }
+        };
+    }
+
+    @Override
+    public IndexMap getIndexerMap() {
+        return new IndexMap() {
 
             @Override
-            public EncodedSpeedController getLeft() {
-                final SendableSpeedController leftGroup = SendableSpeedController.group(new WPI_TalonSRX(1),
-                        new WPI_TalonSRX(4));
-                return EncodedSpeedController.join(leftGroup, new MockEncoder());
+            public SendableSpeedController singulator() {
+                final Victor indexingMotor = new Victor(4);
+                return SendableSpeedController.wrap(indexingMotor);
+            }
+
+            @Override
+            public SendableSpeedController pierreMotor() {
+                final Victor pierreMotor = new Victor(5);
+                return SendableSpeedController.wrap(pierreMotor);
+            }
+            
+            public BooleanSupplier frontIntakeIR() {
+                AnalogTrigger frontIntakeIR = new AnalogTrigger(0);
+                frontIntakeIR.setLimitsVoltage(1.2,1.4);
+                return frontIntakeIR::getTriggerState;
             }
+
+            public BooleanSupplier bottomPierreIR() {
+                AnalogTrigger bottomPierreIR = new AnalogTrigger(1);
+                bottomPierreIR.setLimitsVoltage(1.2, 1.4);
+                return bottomPierreIR::getTriggerState;            }
```

Comment:
Not sure why this file didn't autoformat, the `}` should be on a new line.

Reply:
It didn't autoformat because he disabled format on save...